test/integration/cli-test.coffee

```diff
@@ -4,6 +4,9 @@ express = require 'express'
 clone = require 'clone'
 bodyParser = require 'body-parser'
 fs = require 'fs'
+syncExec = require 'sync-exec'
```

Comment:
This would use native implementation if available (node >= 0.12):

```coffee
execSync = require('child_process').execSync or require('sync-exec')
```